doc/tutorial-basic.md

```diff
@@ -76,14 +76,14 @@ Use case: how much of the read content is contained in the reference genome?
 Build a signature for an E. coli genome:
 
 ```
-sourmash compute --scaled 1000 -k 31 ~/data/ecoliMG1655.fa.gz -o ecoli-genome.sig
+sourmash sketch dna -p scaled=1000,k=31 ~/data/ecoliMG1655.fa.gz -o ecoli-genome.sig
 ```
 
 and now evaluate *containment*, that is, what fraction of the read content is
 contained in the genome:
 
 ```
-sourmash search -k 31 ecoli-reads.sig ecoli-genome.sig --containment
+sourmash search 31 ecoli-reads.sig ecoli-genome.sig --containment
```

Comment:
um is this correct? It feels weird and naked with no `k`...

Reply:
indeed it is not ;). fixed!